### PR TITLE
Baseline model

### PR DESCRIPTION
**Pull request by nicholasshort**
Added baseline model. Current average accuracy over 100 test samples is 55%

### CODE REVIEW

**Comment by jack-jintao-huang** [comment]
Please merge this pr with the current main

File: src/baseline/preprocess.py
Comment (terryluan12) [inline src/baseline/preprocess.py:1]: Should this be moved into the `src/preprocessing` module? Also, it'd probably be a good idea to put the code into a `preprocess_knn` function.

Comment (nicholasshort) [reply src/baseline/preprocess.py:1]: Sure good idea

Comment (nicholasshort) [reply src/baseline/preprocess.py:1]: @terryluan12 Done.

**Review by tmahabir — APPROVED** [review]
👍🏽

File: src/baseline/preprocess.py
Comment (nicholasshort) [reply src/baseline/preprocess.py:1]: Actually, I mistakenly named the file `preprocessing`. In actuality, all it does is call a a few preprocessing functions to return appropriate dataloaders. For this reason, I have moved the file back to the baseline directory and renamed it to `datasources`.

**Review by terryluan12 — APPROVED** [review]

**Pull request merged**